spec/integration/outputs/retry_spec.rb

```diff
@@ -22,7 +32,7 @@ def mock_actions_with_response(*resp)
       "template_overwrite" => true,
       "hosts" => get_host(),
       "port" => get_port(),
-      "retry_max_items" => 10,
```

Comment:
and this should be returned I think. not sure it changes much since this test is based on time... but no reason to change it in this PR I feel